### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by masa213f**
I want to update documents one by one.

Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,12 +16,19 @@ clusters.
 
 ## Features
 
-- Self-hosted runner pool
-  - Run a time-consuming startup script before a job starts
-- Notification
-  - Users can notice that a job succeeds or not with notification
-- Workflow failure investigation
-  - Users can extend the lifetime of a `Pod` and investigate what causes the failure
+- Run a self hosted runner on a pod
```
Comment (zoetrope) [inline README.md:19]: orthographic variants
```suggestion
- Run a self-hosted runner on a pod
```

**Comment by kmdkuk** [comment]
There is no uniformity in whether the word "meows" is enclosed in backquotes or not, but this is a minor detail.
Other than that, nothing much to say.

**Review by zoetrope — APPROVED** [review]
LGTM

**Review by yokaze — APPROVED** [review]
LGTM

**Comment by masa213f** [comment]
@kmdkuk 
I'm enclosing the first `meows` to emphasize because it is the first definition of the software name.
I don't think I need to emphasize the other parts.

**Review by kmdkuk — APPROVED** [review]
LGTM

**Pull request merged**